modules/table/src/main/java/org/apache/ignite/internal/table/distributed/replication/request/BuildIndexReplicaRequest.java

```diff
@@ -19,11 +19,16 @@
 
 import java.util.List;
 import java.util.UUID;
+import org.apache.ignite.internal.replicator.exception.PrimaryReplicaMissException;
 import org.apache.ignite.internal.replicator.message.ReplicaRequest;
 import org.apache.ignite.internal.table.distributed.TableMessageGroup;
 import org.apache.ignite.network.annotations.Transferable;
 
-/** Replica request to build a table index. */
+/**
+ * Replica request to build a table index.
+ *
+ * <p>It is possible to receive an {@link PrimaryReplicaMissException} in response to message processing if the leaseholder changes.</p>
```

Comment:
```suggestion
 * <p>It is possible to receive a {@link PrimaryReplicaMissException} in response to message processing if the leaseholder changes.</p>
```

Reply:
Is this addition really relevant? I mean, you can receive that exception for all replica requests, right? I don't understand the placement of this comment in this particular class.

Reply:
I corrected the typo and discussed it personally and decided to leave it as is.